css/geometry-1/DOMMatrix-css-string.worker.js

```diff
@@ -0,0 +1,18 @@
+// https://drafts.fxtf.org/geometry/#DOMMatrix
+
+importScripts("/resources/testharness.js");
+
+['DOMMatrix', 'DOMMatrixReadOnly'].forEach(constr => {
+  test(() => {
+    assert_true(constr in self, `${constr} is not supported`);
```

Comment:
When this fails, it will be "assert_true: DOMMatrix is not supported expected true got false". In other words, the description is inverted, if it describes the expected value (maybe "self.${constr} exists") then the message will make sense. Or just no description since there's just one assert_true in the test.